### PR TITLE
PXB-2709 Create 8.0.28 release notes and release note tasks 

### PR DESCRIPTION
**Pull request by alina-derkach-oaza**
modified: source/conf.py
modified: source/release-notes.rst
new file: source/release-notes/8.0/8.0.28-20.0.rst

### CODE REVIEW

**Review by patrickbirch — APPROVED** [review]
Good

File: storage/innobase/xtrabackup/doc/source/release-notes/8.0/8.0.28-20.0.rst [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+:Installation: `Installing Percona XtraBackup <https://www.percona.com/doc/percona-xtrabackup/8.0/installation.html>`_
+
+`Percona XtraBackup <https://www.percona.com/software/mysql-database/percona-xtrabackup>`_ enables MySQL backups without blocking user queries, making it ideal for companies with large data sets and mission-critical applications that cannot tolerate long periods of downtime. Offered free as an open source solution, it drives down backup costs while providing unique features for MySQL backups.
+
+Release Highlights
+=================================================
+
+The following list contains a number of the notable updates and fixes for *MySQL* 8.0.28, provided by Oracle, and included in Percona XtraBackup for MySQL:
+
+* The GnuPG build key has been updated. The GnuPG build key is used to sign MySQL downloadable packages. The systems configured to use repo.mysql.com may report a signature verification error when upgrading using *apt* or *yum* to MySQL 5.7.37 and higher or to MySQL 8.0.28 and higher. To resolve this issue, use one of the following methods:
```
Comment (altmannmarcelo) [inline storage/innobase/xtrabackup/doc/source/release-notes/8.0/8.0.28-20.0.rst]: This is not relevant to PXB

File: storage/innobase/xtrabackup/doc/source/release-notes/8.0/8.0.28-20.0.rst [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+Find more information about DDL operations that support `ALGORITHM=INSTANT` in `Online DDL Operations <https://dev.mysql.com/doc/refman/8.0/en/innodb-online-ddl-operations.html>`__.
+
+Find the full list of bug fixes and changes in the `MySQL 8.0.28 Release Notes <https://dev.mysql.com/doc/relnotes/mysql/8.0/en/news-8-0-28.html>`__.
+
+Bugs Fixed
+==================================================
+
+* :jirabug:`PXB-2662`: Percona XtraBackup exited during prepare when creating an incremental backup using page tracking. 
+* :jirabug:`PXB-1676`: The size of the redo log was incorrect after creating a backup.
```
Comment (altmannmarcelo) [inline storage/innobase/xtrabackup/doc/source/release-notes/8.0/8.0.28-20.0.rst]: This is misleading. The issue is about fix an error message only.

File: storage/innobase/xtrabackup/doc/source/release-notes/8.0/8.0.28-20.0.rst [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+Find more information about DDL operations that support `ALGORITHM=INSTANT` in `Online DDL Operations <https://dev.mysql.com/doc/refman/8.0/en/innodb-online-ddl-operations.html>`__.
+
+Find the full list of bug fixes and changes in the `MySQL 8.0.28 Release Notes <https://dev.mysql.com/doc/relnotes/mysql/8.0/en/news-8-0-28.html>`__.
+
+Bugs Fixed
+==================================================
+
+* :jirabug:`PXB-2662`: Percona XtraBackup exited during prepare when creating an incremental backup using page tracking. 
+* :jirabug:`PXB-1676`: The size of the redo log was incorrect after creating a backup. 
+* :jirabug:`PXB-2701`: An incremental backup on MyRocks did not determine if the previous full or incremental backup had the same files.
```
Comment (altmannmarcelo) [inline storage/innobase/xtrabackup/doc/source/release-notes/8.0/8.0.28-20.0.rst]: This is a documentation bug. We should remove it from release notes as they are not tied to a specific release.

**Review by altmannmarcelo — CHANGES_REQUESTED** [review]
We are missing a few key fixes:

PXB-2716 - With especial thanks to @Bo98 for fixing it.
PXB-2714
PXB-2670

I'm not sure the part of highlights for upstream has much to do with PXB - It will make more sense to highlight them on PS as they will affect it directly.

There is also some work that will most likely be part of this release
https://jira.percona.com/browse/PXB-2721
https://jira.percona.com/browse/PXB-1818

File: storage/innobase/xtrabackup/doc/source/release-notes/8.0/8.0.28-20.0.rst (hunk 1 above)
Comment (patrickbirch) [reply storage/innobase/xtrabackup/doc/source/release-notes/8.0/8.0.28-20.0.rst]: Yes. We'll remove this and add information about PXB.

**Review by george-lorch — APPROVED** [review]

File: storage/innobase/xtrabackup/doc/source/release-notes/8.0/8.0.28-20.0.rst (hunk 3 above)
Comment (alina-derkach-oaza) [reply storage/innobase/xtrabackup/doc/source/release-notes/8.0/8.0.28-20.0.rst]: Removed

File: storage/innobase/xtrabackup/doc/source/release-notes/8.0/8.0.28-20.0.rst (hunk 2 above)
Comment (alina-derkach-oaza) [reply storage/innobase/xtrabackup/doc/source/release-notes/8.0/8.0.28-20.0.rst]: Fixed

File: storage/innobase/xtrabackup/doc/source/release-notes/8.0/8.0.28-20.0.rst (hunk 1 above)
Comment (alina-derkach-oaza) [reply storage/innobase/xtrabackup/doc/source/release-notes/8.0/8.0.28-20.0.rst]: Updated

**Review by altmannmarcelo — APPROVED** [review]
LGTM

**Review by patrickbirch — APPROVED** [review]

**Review by george-lorch — APPROVED** [review]
+1

**Pull request merged**